YangMingShan/YMSPhotoPicker/Private/YMSAlbumPickerViewController.m

```diff
@@ -95,11 +95,6 @@ - (void)viewDidLayoutSubviews
     }
 }
 
-- (UIStatusBarStyle)preferredStatusBarStyle
-{
-    return [YMSPhotoPickerTheme sharedInstance].statusBarStyle;
```

Comment:
Why removing this? If the navigation bar background color is dark, this setting is useful.

Reply:
Because the latest version put YMSAlbumPickerViewController under YMSNavigationController, the preferredStatusBarStyle is set by YMSNavigationController now.